gensim/models/word2vec.py

```diff
@@ -1090,6 +1090,13 @@ def update_weights(self):
         for i in xrange(len(self.wv.syn0), len(self.wv.vocab)):
             # construct deterministic seed from word AND seed argument
             newsyn0[i-len(self.wv.syn0)] = self.seeded_vector(self.wv.index2word[i] + str(self.seed))
+
+        # Raise an error if an online update is run before initial training on a corpus
+        if not len(self.wv.syn0):
+            raise RuntimeError("You can do an online update of vocabulary on a pre-trained model. " \
```

Comment:
Please start the message by stating the cause of the warning: "The model has not yet been trained."